src/lib/Strategy/NotificationTwigStrategy.php

```diff
@@ -53,6 +53,8 @@ public function setDefault(string $defaultTemplate)
      */
     public function decide($contentId): string
```

Comment:
this should be prolly handed somewhere else, but afair those are decoded from json, right?

Reply:
Actually should be resolved as `int $contentId` as passing anything other than int is incorrect. But maybe out of scope, so can be handled as a follow-up as I've mentioned somewhere else.

Reply:
> But maybe out of scope, so can be handled as a follow-up as I've mentioned somewhere else.

I agree.